lib/active_decorator/rspec.rb

```diff
@@ -15,7 +15,11 @@ def self.enable(example)
                      ActionController::Base
                    end
       controller = Class.new(base_class).new
-      controller.request = ActionController::TestRequest.new
+      controller.request = if ActionPack::VERSION::STRING >= '5'
```

Comment:
I prefer `ActionController::TestRequest.respond_to?(:create)` to `ActionPack::VERSION::STRING >= '5' `.  But how about you?